src/main/java/org/jlibvips/jna/VipsBindingsSingleton.java

```diff
@@ -18,11 +18,20 @@ public static VipsBindings instance() {
             if(libraryPath == null || libraryPath.isEmpty()) {
                 throw new IllegalStateException("Please call VipsBindingsSingleton.configure(...) before getting the instance.");
             }
-            INSTANCE = Native.load(libraryPath, VipsBindings.class);
+            try {
+                INSTANCE = Native.load(libraryPath, VipsBindings.class);
+            } catch (UnsatisfiedLinkError e) {
+                libraryPath = guessPath();
+                INSTANCE = Native.load(libraryPath, VipsBindings.class);
+            }
         }
         return INSTANCE;
     }
 
+    private static String guessPath() {
+        return "/usr/local/Cellar/vips/8.10.2_4/lib/libvips.dylib";
```

Comment:
This is a hack to make the tests run on my Mac, with libvips installed using homebrew (`brew install vips`). Ideally the tests should detect OS, then try to find the library in some common places, or look in a sysprop or env.var for the location.

Reply:
I implmemented environment variable support instead of this hardcoded hack